app/controllers/categories_controller.rb

```diff
@@ -30,7 +30,8 @@ def index
       is_homepage: current_homepage == "categories",
       parent_category_id: params[:parent_category_id],
       include_topics: include_topics(parent_category),
-      include_subcategories: include_subcategories
+      include_subcategories: include_subcategories,
+      tag: params[:tag]
```

Comment:
From an API perspective, I wonder if it makes more sense to just support an array of tags instead of just a single tag.

Reply:
I would prefer to keep this as simple as possible to start. It would be easy to expand upon this later and we have not had (any / very few) requests for this functionality... so keeping it bare bones to start is best imo

Reply:
> It would be easy to expand upon this later

Not really because you would need to support a `tags` params instead and we end up with `tag` and `tags` params which I feel is unnecessary when a single `tags` param will suffice for both use cases.